packages/frontend/core/src/blocksuite/extensions/link-preview-service/index.ts

```diff
@@ -0,0 +1,10 @@
+import type { FrameworkProvider } from '@toeverything/infra';
+
+import { patchLinkPreviewCache } from './link-preview-cache';
+import { patchLinkPreviewService } from './link-preview-service';
+
+export const patchLinkPreviewServiceExtension = (
```

Comment:
This pattern is not recommended. You can take a look at [here to know how to expose the config](https://github.com/toeverything/AFFiNE/blob/canary/packages/frontend/core/src/blocksuite/extensions/theme/index.ts), and [here to know how to extend the config builder](https://github.com/toeverything/AFFiNE/blob/canary/packages/frontend/core/src/blocksuite/manager/migrating-view.ts#L56)